### PR TITLE
Added plugins page list view

### PR DESCRIPTION
**Pull request by jacobcoughenour**

### CODE REVIEW

File: _layouts/plugins.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 <i class="listicon fa fa-th-list" aria-hidden="true"></i>
             </label>
         </div>
-        <div id="plugins-index">
+        <div id="plugins-index" style="opacity: 0;">
```
Comment (lukespragg) [inline _layouts/plugins.html]: Tsk tsk, inline? :P

File: _layouts/plugins.html [hunk 2]
```diff
@@ -64,12 +64,35 @@ <h3 class="grid-view-elem-title">
 </div>
 <script type="text/javascript">
     $(document).ready(function () {
+        let index = $("#plugins-index");
```
Comment (lukespragg) [inline _layouts/plugins.html]: JavaScript should be moved to the custom.js file.

File: assets/css/main.css [hunk 3]
```diff
@@ -18442,3 +18438,54 @@ input.changelog-input {
...
+}
+.list-mode .plugin-card .plugin-info .grid-view-elem-title {
+    margin-bottom: 0;
+}
+.list-mode .plugin-card .plugin-info .grid-view-elem-description {
+    padding-bottom: 0;
+}
+.list-mode .plugin-card .plugin-info img.badge {
+    display: none;
+}
```
Comment (lukespragg) [inline assets/css/main.css]: Don't remove the end-of-file new lines please.

File: _layouts/plugins.html (hunk 2 above)
Comment (jacobcoughenour) [reply _layouts/plugins.html]: This javascript is specific to the plugins page,  I don't see why it needs to be loaded on every page.  I'm just following how there is javascript at the end of index.html

Comment (jacobcoughenour) [reply _layouts/plugins.html]: Maybe it should be it's own .js file.

**Comment by lukespragg** [comment]
Merged. :)

**Pull request closed without merging**